### ISSUE TITLE
Full WYSIWYG converts urls

### ISSUE THREAD

**Issue by rijkvanzanten** [directus/directus#1896]
_Originally mentioned by @flo-f in #1773_

## Version Info
  - Directus version and branch: develop 5ce02d371465e8ce07e634453f7319ef6cd4c066

## Behavior
TinyMCE is replacing my URLs: I inserted absolute URLs to my Directus storage folder and TinyMCE rewrote these to relative URLs.

I entered `https://www.mydomain.com/directus/storage/...` which was replaced by `../../storage/...`

**Comment by rijkvanzanten** [comment]
Temp fix by @flo-f:

As I needed the URLs to remain absolute, I modified the file [`/app/core/interfaces/wysiwyg_full/interface.js`](https://github.com/directus/directus/blob/598fa1a06ebcc5ecbb5c94ef9ac85e03f550e140/app/core/interfaces/wysiwyg_full/interface.js) by adding a fixed option `convert_urls` right above TinyMCE initialization (currently close to line 210 of `interface.js`):

```javascript
options.convert_urls = false;
tinyMCE.init(options); // this line already exists - don't duplicate ;-) 
```

**Comment by rijkvanzanten** [comment]
This has been fixed by 71abff9f6d4503ad0e0c005894461eebb6cbedf1 on the [`develop`](https://github.com/directus/directus/tree/develop) branch and will release soon.

**Comment by flo-f** [comment]
Thank you very much for the immediate fix!
